app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -305,6 +304,16 @@ def handle_expired_status_update(enrollment, response, response_message)
     end
   end
 
+  def handle_fraud_review_pending(enrollment)
+    return unless IdentityConfig.store.in_person_proofing_enforce_tmx
```

Comment:
So this is totally a super nit, but I'm wondering if it makes sense to move this config check into it's own method like we do for `ipp_enabled` on line 65 and then use it on line 432 to determine if we should call `handle_fraud_review_pending`? I kinda like that pattern of escalating and making more explicit the prerequisites for a method to be called. But also, this is non-blocking I think.